diesel_compile_tests/tests/fail/numeric_ops_require_numeric_column.stderr

```diff
@@ -13,3 +13,6 @@ note: an implementation of `std::ops::Add` might be missing for `columns::name`
    |         ^^^^ must implement `std::ops::Add`
 note: the trait `std::ops::Add` must be implemented
   --> $RUST/core/src/ops/arith.rs
+   |
+   | pub trait Add<Rhs = Self> {
```

Comment:
ah it looks like this may be due to differing versions of Rust between CI and version that was used to generate the snapshots